src/v2/stack_aggregator.py

```diff
@@ -212,14 +183,7 @@ def _get_package_details(self, component):
         ecosystem, pkg = _get_pkg_from_graph_version_node(version_node)
         github_details = _get_github_details(pkg_node)
         public_vulns, private_vulns = self._get_vulnerabilities(component.get("vuln", {}))
-        recommended_latest_version = None
```

Comment:
@arajkumar Plz confirm. This latest version is not the latest non cve version and not the latest version for the pkg.. right?

Reply:
Yes, we have two fields, `latest_version` & `recommended_latest_version`. The former one always points to the latest release of a particular package & the latter one points to the non cve recent version.